Sources/PasteBoardDependency/PasteboardDependency_iOS.swift

```diff
@@ -0,0 +1,425 @@
+                getImages: .init {
+                    UIPasteboard.general.images
+                },
+                setColor: .init { color in
+                    UIPasteboard.general.color = color
+                },
+                getColor: .init {
+                    UIPasteboard.general.color
+                },
+                setColors: .init { colors in
+                    UIPasteboard.general.colors = colors
+                },
+                getColors: .init {
+                    UIPasteboard.general.colors
+                },
+                contains: .init { types in
+                    UIPasteboard.general.contains(pasteboardTypes: types)
+                },
+                itemSet: .init { types in
+                    UIPasteboard.general.itemSet(withPasteboardTypes: types)
+                },
+                setItems: .init { items in
+                    UIPasteboard.general.items = items
+                },
+                items: .init {
+                    UIPasteboard.general.items
+                },
+                numberOfItems: .init {
+                    UIPasteboard.general.numberOfItems
+                },
+                addItems: .init { items in
+                    UIPasteboard.general.addItems(items)
+                },
+                setItemsWithOptions: .init { items, options in
+                    UIPasteboard.general.setItems(items, options: options)
+                },
+                dataForPasteboardType: .init { type in
+                    UIPasteboard.general.data(forPasteboardType: type)
+                },
+                dataForPasteboardTypeInItemSet: .init { type, itemSet in
+                    UIPasteboard.general.data(forPasteboardType: type, inItemSet: itemSet)
+                },
+                setData: .init { data, type in
+                    UIPasteboard.general.setData(data, forPasteboardType: type)
+                },
+                valueForPasteboardType: .init { type in
+                    UIPasteboard.general.value(forPasteboardType: type)
+                },
+                valuesForPasteboardTypeInItemSet: .init { type, itemSet in
+                    UIPasteboard.general.values(forPasteboardType: type, inItemSet: itemSet)
+                },
+                setValueForPasteboardType: .init { value, type in
+                    UIPasteboard.general.setValue(value, forPasteboardType: type)
+                },
+                types: .init {
+                    UIPasteboard.general.types
+                },
+                typesForItemSet: .init { itemSet in
+                    UIPasteboard.general.types(forItemSet: itemSet)
+                },
+                hasColors: .init {
+                    UIPasteboard.general.hasColors
+                },
+                hasImages: .init {
+                    UIPasteboard.general.hasImages
+                },
+                hasStrings: .init {
+                    UIPasteboard.general.hasStrings
```

Comment:
I think you actually want to check whether it's non-nil for this reason: `https://stackoverflow.com/a/66041472/18358392`. At least, that's what I'm doing in my local `PasteboardDependency` implementation.

Reply:
Ah, then again...this is a dependency wrapper so that suggestion probably doesn't apply here.